site/src/components/srv-settings/emotes.js

```diff
@@ -168,7 +168,6 @@ const component = (state, emit) => {
     </table>
 
     <div class='submit'>
-      <span class='status'></span>
```

Comment:
is this intentional??

Reply:
Yes: https://github.com/towerofnix/decent/pull/141/commits/b2aab1f9882ac4f2232ade657bc0b7dc2fe8b99c

That `status` element isn't used anywhere in `emotes.js`?

(Admittedly I didn't test it, so if there's an error, I wouldn't know. Oops.)

Reply:
There was no error when I (just) tested adding an emoji.

Reply:
Oh, I guess you're right.

I seem to remember it having a save/status feature, but clearly not - I think it's because I moved it to using modals.